applications/asset_tracker_v2/doc/asset_tracker_v2_description.rst

```diff
@@ -234,6 +234,11 @@ Check and configure the following configuration options for the application:
 CONFIG_ASSET_TRACKER_V2_APP_VERSION - Configuration for providing the application version
    The application publishes its version number as a part of the static device data. The default value for the application version is ``0.0.0-development``. To configure the application version, set :ref:`CONFIG_ASSET_TRACKER_V2_APP_VERSION <CONFIG_ASSET_TRACKER_V2_APP_VERSION>` to the desired version.
 
+.. _CONFIG_GNSS_MODULE_ANTENNA_EXTERNAL:
```

Comment:
Is it necessary to list this option here when its already listed in `gnss_module.rst` ? Can it not just be referred to?

Reply:
Good point. Maybe it is enough that it is in one place only. @umapraseeda thoughts?

Reply:
Yes, could you try linking to that definition in gnss_module.rst @peknis ? That should work.

Reply:
Seems to be enough to have a definition for the option in one place only.